content/applications/finance/accounting/others/adviser/budget.rst

```diff
@@ -2,156 +2,88 @@
 
-Repeat this steps to create a revenue budgetary position. Only in this
-case select the relevant income accounts.
+.. image::  budget/smith-and-co-expenses.png
+  :align: center
 
-Analytical account
-------------------
+Now, let's repeat the steps to create a budgetary position that reflects the revenue.
 
-Odoo needs to know which costs or
-expenses are relevant to a specified budget. To do so we need to link
-our invoices and expenses to a defined analytical account. Create an
-analytical account by entering the Accounting module and clicking
-:menuselection:`Advisers --> Analytic Accounts --> Open Charts`. Create a new Account
-called Smith&Co project and select the related partner.
+.. image::  budget/smith-and-co-revenue.png
+  :align: center
 
-.. image:: budget/budget06.png
-   :align: center
-
-Set a budget
-============
+Analytical accounts
+-------------------
 
-Let's now set our targets for our budget. We specified that we expect to
-gain 1000 with this project and we would like not to spend more than
-700.
+Odoo needs to know which costs or expenses are relevant to a specified budget because the above
+general accounts may be used for different projects. To do so we need to link our invoices and
+expenses to a defined analytical account: Go to :menuselection:`Accounting --> Configuration -->
+Analytic Accounts` and :guilabel:`Create` a new :guilabel:`Analytic Account` called Smith & Co.
 
-To set those targets, enter the accounting app, select :menuselection:`Advisers -->
-Budgets` and create a new Budget.
+Each budgetary position can have any number of accounts from the general ledger (the main chart of
+accounts) assigned to it, though it must have at least one.
 
-We have to give a name to the budget. In this case we'll call it "Smith
-Project". Select the period wherein the budget will be applicable. Next
-add an item to specify your targets in the Budget Line.
+Define the budget
+-----------------
 
-.. image:: budget/budget07.png
-   :align: center
+Let’s now set our targets. We specified that we expect to gain 1000 with this project and we would
+like not to spend more than 700. Go to :menuselection:`Accounting --> Budgets` and
+:guilabel:`Create` a new budget.
 
-Select the Budgetary Position related to the Budget Line. In other
-words, select the position that points to the accounts you want to
-budget. In this case we will start with our 700 maximum charge target.
-Select the "Cost" Budgetary Position and specify the Planned Amount.
-As we are recording a cost, we need to specify a **negative amount**.
-Finally, select the corresponding analytic account.
+First, fill in your :guilabel:`Budget Name`. Then, select the :guilabel:`Period` wherein the budget
+is applicable. Next, add the :guilabel:`Budgetary Position` you want to track, define the related
+:guilabel:`Analytic Account`, and add the :guilabel:`Planned Amount`.
 
-.. image:: budget/budget08.png
+.. image:: budget/define-the-budget.png
    :align: center
 
```

Comment:
Do we let it as a draft, confirm, approve, done? What are these status used for?

Reply:
good point. Clarifications ongoing.

Reply:
as per discussion with MALB, better to skip this part in the doc as it is probably not up to date...